Makefile

```diff
@@ -2,6 +2,8 @@ GO ?= $(shell command -v go 2> /dev/null)
 NPM ?= $(shell command -v npm 2> /dev/null)
 CURL ?= $(shell command -v curl 2> /dev/null)
 MANIFEST_FILE ?= plugin.json
+GO_TEST_FLAGS ?= -race
+GO_BUILD_FLAGS ?=
```

Comment:
Added this here to make clear what the comment of the variable is. 1/5

Reply:
sorry, to make clear which comment?

Reply:
Dump type. Please take another look.